app/model/blockchain/token.py

```diff
@@ -199,12 +189,9 @@ def get(session, token_address: str):
         personal_info_address = TokenContract.functions.personalInfoAddress().call()
 
         # 企業リストから、企業名を取得する
-        company_name = ""
-        rsa_publickey = ""
-        for company in company_list:
-            if to_checksum_address(company['address']) == owner_address:
-                company_name = company['corporate_name']
-                rsa_publickey = company['rsa_publickey']
+        company = CompanyList.get_find(to_checksum_address(owner_address))
+        company_name = company.corporate_name
+        rsa_publickey = company.rsa_publickey
```

Comment:
I think you should set a default value if the data does not exist (None).

Reply:
If not exists, 'get_find' returns blank Company object(created with CompanyList.DEFAULT).
In that case, Token set the same blank as before change.